tests/e2e/tidbcluster/stability.go

```diff
@@ -119,8 +127,6 @@ var _ = ginkgo.Describe("[tidb-operator][Stability]", func() {
 		ginkgo.AfterEach(func() {
 			ginkgo.By("Uninstall tidb-operator")
 			oa.CleanOperatorOrDie(ocfg)
-			ginkgo.By("Uninstalling CRDs")
-			oa.CleanCRDOrDie()
```

Comment:
don't delete CRDs, otherwise we can't keep test pods with `--delete-namespace=false`.